src/index.js

```diff
@@ -20,27 +25,71 @@ const propertyValueConverters = {
     if (isNumber(value)) {
       return value
     }
-    const splitValues = value.replace(/ +/g, ' ').split(' ')
-    if (splitValues.length <= 3) {
-      return value
-    }
-    const [top, right, bottom, left, important] = splitValues
-    if (important && important !== '!important') {
-      return value // the value is invalid and we can't do anything about that...
-    }
-    return [top, left, bottom, right, important].filter(Boolean).join(' ')
+    return handleQuartetValues(value)
   },
   textShadow(value) {
     // intentionally leaving off the `g` flag here because we only want to change the first number (which is the offset-x)
-    return value.replace(/(-*)(\d+)/, (match, negative, number) => {
+    return value.replace(/(-*)([.|\d]+)/, (match, negative, number) => {
       if (number === '0') {
         return match
       }
       const doubleNegative = negative === '' ? '-' : ''
       return `${doubleNegative}${number}`
     })
   },
+  borderColor(value) {
+    return handleQuartetValues(value)
+  },
+  borderRadius(value) {
+    if (isNumber(value)) {
+      return value
+    }
+    if (includes(value, '/')) {
+      const [radius1, radius2] = value.split('/')
+      const convertedRadius1 = propertyValueConverters.borderRadius(radius1.trim())
+      const convertedRadius2 = propertyValueConverters.borderRadius(radius2.trim())
+      return `${convertedRadius1} / ${convertedRadius2}`
+    }
+    const splitValues = getValuesAsList(value)
+    switch (splitValues.length) {
+      case 2:
+        return splitValues.reverse().join(' ')
+      case 4: // eslint-disable-line no-case-declarations
```

Comment:
Can you wrap this in `{}`s to get rid of this warning? Like

```
case 4: {
    const ...
    return ...
}
default:
```

(since `const`s are block scoped)